frontend/src/scenes/surveys/surveyLogic.tsx

```diff
@@ -1006,13 +1052,14 @@ export const surveyLogic = kea<surveyLogicType>([
                                 operator: PropertyOperator.Exact,
                                 value: survey.id,
                             },
+                            ...propertyFilters,
                         ],
                     },
                     propertiesViaUrl: true,
                     showExport: true,
                     showReload: true,
                     showEventFilter: false,
-                    showPropertyFilter: true,
+                    showPropertyFilter: false,
```

Comment:
is that intentional?

Reply:
yes, to not show the events filters directly on the `<Query>` component, as we moved it directly to the Survey component